tools/test/examples/examples_lib.py

```diff
@@ -19,7 +19,8 @@
 from tools.export import EXPORTERS
 
 SUPPORTED_TOOLCHAINS = ["ARM", "IAR", "GCC_ARM"]
-SUPPORTED_IDES = ["iar", "uvision", "make_gcc_arm", "make_iar", "make_armc5"]
+SUPPORTED_IDES = [exp for exp in EXPORTERS.keys() if exp != "cmsis" and exp != "zip"]
```

Comment:
Further, their generate functions are just bizarre. Zip returns True and CMSIS exports a useless .cpdsc (this one is ust experimental right now).

If we did that, then those others that don't have a build (kds, etc) wouldn't be allowed to do the just exporting part (which is the point of this pr).